skimage/morphology/tests/test_ccomp.py

```diff
@@ -66,10 +67,18 @@ def test_diag(self):
     def test_4_vs_8(self):
         x = np.array([[0, 1],
                       [1, 0]], dtype=int)
-        assert_array_equal(label(x, 4),
+        with expected_warnings(["use 'connectivity'"]):
+            assert_array_equal(label(x, 4),
```

Comment:
maybe put `ǹeighbors=4` here?

Reply:
It's hard to tell the original meaning of the code. Maybe the test was designed to ensure positional arguments work as expected